README.md

```diff
@@ -1,2 +1,213 @@
 # bootstrap
+
 Foundation Bootstrap Team
+
+## About this document
+
+The Node.js and JS Foundations have announced an intent to merge into a single
+umbrella organization hosting the combined portfolio of projects currently
+under each separate organization.
+
+The Board's of each Foundation have asked the various projects to come together
+and provide specific feedback on:
+
+1. Whether the merger should proceed
+2. What the overall structure of a merged Foundation should look like
+3. How projects should be organized within a merged Foundation
+4. What representation projects should have on a merged Foundation's Board of
+   Directors
+
+To help with the process of organizing, drafting, and submitting the requested
+feedback, the technical projects under both the Node.js and JS Foundations are
+forming a temporary body (the "Node+JS Bootstrap Committee").
+
+This document, along with the other files in this repository, are living
+documents where the Committee will capture and prepare the feedback requested
+by the Boards. The documents here will not represent the finalized governance
+model of a merged Foundation but will serve as input into the generation of
+new Foundation Bylaws and Top Level Project Charters should the merger of the
+Foundations move forward.
+
+Until such time that this group has reached consensus on a new governance
+model, and the Foundation has ratified new By-laws and charters, all projects,
+programs, and committees will continue to operate unchanged.
+
+Upon enactment of a new merged Foundation, or upon decision that the merger
+should not proceed, this working body will be disbanded.
+
+### Contributing to this repository
+
+Anyone may contribute to this repository via pull request. All individuals
+holding commit rights to any Node.js Foundatioon or JS Foundation project may
+participate in the Node+JS Bootstrap Committee and may request commit access
+to this repository.
```

Comment:
I think we should have an explicit +1 from someone representing the JSF for this. I don't see any conflicts, but would be good to have explicit buy in

perhaps @dmethvin or jorydotcom